### ISSUE TITLE
Inquiry on the purpose on the normalization of SLAM camera 6D angular velocity

### ISSUE THREAD

**Issue by GaigeY** [yohanshin/WHAM#95]
Hi @yohanshin ,

Sorry to bother you again. I'm meeting another confusing problem in SLAM result.

I noticed that you've normalized the camera 6D poses [here](https://github.com/yohanshin/WHAM/blob/2b54f7797391c94876848b905ed875b154c4a295/lib/data/datasets/dataset_custom.py#L35):
``` python
cam_angvel = cam_angvel - torch.tensor([[1, 0, 0, 0, 1, 0]]).to(cam_angvel) # Normalize
```
This is a seldom seen treatment, as I have rarely seen anyone subtract the angle of rotation from the unit vector, and your AMASS CameraAugmentor [here](https://github.com/yohanshin/WHAM/blob/2b54f7797391c94876848b905ed875b154c4a295/lib/data/utils/augmentor.py#L226) didn't apply this normalization strategy. May I ask if you have any basis for doing so? Or is this an empirical strategy for dealing with this?

Best wishes,

Qijun Ying

**Comment by GaigeY** [comment]
I discover that you've made same normalization to the AMASS synthetic cam_angvel [here](https://github.com/yohanshin/WHAM/blob/2b54f7797391c94876848b905ed875b154c4a295/lib/data/utils/augmentor.py#L286), too.
